__tests__/api.test.js

```diff
@@ -0,0 +1,42 @@
+const db = require('../db/connection');
+// ref object = const ____ = require('../utilities');
+const { categoryData, commentData, reviewData, userData } = require('../db/data/test-data');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const request = require('supertest');
+
+
+beforeEach(()=> seed({ categoryData, commentData, reviewData, userData }))
+
+afterAll(()=> db.end());
+
+describe('GET /api/categories', () =>{
+    test('200 response', ()=>{
+        return request(app).get('/api/categories').expect(200);
+    })
+    test('200 response, returns an array of objects', ()=>{
+        return request(app).get('/api/categories').expect(200).then(({_body: {categories}})=>{
```

Comment:
I can't find it in the docs buti'm unfamiliar with the `_body` key on response - might be safer to stick with `body`!